test/unit/BCoWPool/BCoWPool.tree

```diff
@@ -0,0 +1,8 @@
+BCoWPool::Finalize
```

Comment:
```suggestion
BCoWPool::finalize
```

Reply:
If I do it this way then the contract has to be named `BCoWPoolfinalize` which isn't as clean